guide/getting-started/hardware.md

```diff
@@ -145,6 +147,26 @@ Bitcoin ATMs are usually bound to local money transmission laws and regulations,
 
 You can find local ATMs using services like [coinatmradar](https://coinatmradar.com/).
 
+## Point of sale
+
+People looking to accept bitcoin payments, especially when using the lightning network, sometimes use dedicated point of sale (POS) hardware terminals when selling goods and services.
+
+{% include picture.html
+   image = "/assets/images/guide/getting-started/hardware/point-of-sale.jpg"
+   retina = "/assets/images/guide/getting-started/hardware/point-of-sale@2x.jpg"
+   mobile = "/assets/images/guide/getting-started/hardware/point-of-sale-mobile.jpg"
+   mobileRetina = "/assets/images/guide/getting-started/hardware/point-of-sale-mobile@2x.jpg"
+   alt-text = "Examples of point of sale hardware devices."
+   width = 800
+   height = 400
+%}
+
+Dedicated bitcoin POS hardware is used to request and receive bitcoin payments in-person. These act very similar to traditional POS systems but present to payees a bitcoin on-chain, lightning, or [unified](https://bitcoinqr.dev/) payment request. Payment requests can also be shared and paid over NFC.
+
+When accepting lightning payments, these POS terminals often connect to a remote always on lightning node. An example of is a merchant using a dedicated iPad to connect to [BTCPay Server](https://btcpayserver.org/) instance.
+
+Accepting lightning payments with POS hardware can also be fully offline using a standard like [LNURL](https://github.com/fiatjaf/lnurl-rfc) to easily receive payments. An example of this is [LNURLPoS](https://github.com/arcbtc/LNURLPoS)
```

Comment:
```suggestion
Accepting lightning payments with POS hardware can also be fully offline, using a standard like [LNURL](https://github.com/fiatjaf/lnurl-rfc) to easily receive payments. An example of this is [LNURLPoS](https://github.com/arcbtc/LNURLPoS).
```
LNURLPoS is a horrible name for a product, hope that will be changed at some point.

Reply:
Agreed on the LNURLPoS thing, but I'd argue it's the name of a technology and not a product. I think Ben is putting the tech out there as FOSS so other people can turn it into an actual product. So my thinking is that any individual or company who picks up his tech will come up with their own unique name for the product.